.github/workflows/tests-rs-package.yml

```diff
@@ -151,3 +160,56 @@ jobs:
           SCCACHE_BUCKET: multi-runner-cache-x1xibo9c
           SCCACHE_REGION: ${{ vars.AWS_REGION }}
           SCCACHE_S3_KEY_PREFIX: ${{ runner.os }}/sccache/${{ steps.arch.outputs.arch }}/linux-gnu
+          
+  verify_features:
+    name: Verify features in ${{ inputs.package }}
+    runs-on: ${{ fromJSON(inputs.test-runner) }}
+    outputs:
+      features: ${{ steps.list.outputs.features }}
+    timeout-minutes: 30
+    if: >
```

Comment:
We should have this condition in tests.yml. This workflow shouldn't know about specific packages